Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -47,7 +48,7 @@ public partial class TextBoxMask
         /// </summary>
         /// <param name="obj">TextBox control</param>
         /// <returns>mask value</returns>
-        public static string GetMask(DependencyObject obj)
+        public static string GetMask(TextBox obj)
```

Comment:
Can you change the sample to ensure that we have use both x:Bind and Binding.. I have a feeling TextBox is going to bust x:Bind